components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -127,37 +113,55 @@ internal class MediaSessionServiceDelegate(
         shutdown()
     }
 
-    private fun processMediaSessionState(state: SessionState?) {
```

Comment:
Could've kept this method but I already need to peek in the `playbackState` in `MediaSessionFeature` to know when to bind the service and I would also prefer to limit the responsibilities of the two parties.
The service/delegate would not duplicate the logic for when to show a notification, they only handle how.